src/Text/Pandoc/App/CommandLineOptions.hs

```diff
@@ -529,9 +529,26 @@ options =
 
     , Option "" ["atx-headers"]
                  (NoArg
-                  (\opt -> return opt { optSetextHeaders = False } ))
+                  (\opt -> do
+                    deprecatedOption "--atx-headers"
+                        "This is now `--markdown-headings=atx`"
```

Comment:
Let's remove the backticks; that is markdown syntax and people
may not know what to make of it in a warning message.